messages/2.0.0

```diff
@@ -2,7 +2,7 @@ SublimeLinter-flow
 ------------------
 * Switched to use flow server
 * Support more syntaxes
-* Now requires flow 0.17.0, please update!
+* Now requires flow 0.19.0, please update!
```

Comment:
Can you revert this change? A new file will need to be created called messages/3.0.0 since this is a breaking change.